helm-chart/renku-graph/values.yaml

```diff
@@ -41,6 +41,8 @@ triplesGenerator:
   ## a demanded number of concurrent triples generation processes
   generationProcessesNumber: 4
   threadsNumber: 6
+  # set this to a pip-installable renku-python version which will be installed on startup
+  renkuPythonDevVersion:
```

Comment:
Would that work if you don't specify a value in your `values.yaml`?

Reply:
this just means it's null